rules/app_clip.bzl

```diff
@@ -0,0 +1,75 @@
+load("@build_bazel_rules_apple//apple:ios.bzl", rules_apple_ios_app_clip = "ios_app_clip")
+load("//rules:plists.bzl", "process_infoplists")
+load("//rules:force_load_direct_deps.bzl", "force_load_direct_deps")
+load("//rules/internal:framework_middleman.bzl", "dep_middleman", "framework_middleman")
+
+def ios_app_clip(name, infoplists_by_build_setting = {}, **kwargs):
+    """
+    Builds and packages an iOS App Clip.
+
+    The docs for app_clip are at rules_apple
+    https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-ios.md#ios_app_clip
+
+    Args:
+        name: The name of the iOS app clip.
+        infoplists_by_build_setting: A dictionary of infoplists grouped by bazel build setting.
+
+                                     Each value is applied if the respective bazel build setting
+                                     is resolved during the analysis phase.
+
+                                     If '//conditions:default' is not set the value in 'infoplists'
+                                     is set as default.
+        **kwargs: Arguments passed to the ios_app_clip rule as appropriate.
+    """
+
+    deps = kwargs.pop("deps", [])
+    frameworks = kwargs.pop("frameworks", [])
+    testonly = kwargs.pop("testonly", False)
+
+    kwargs["families"] = kwargs.pop("families", ["iphone", "ipad"])
```

Comment:
nit: this is required in `rules_apple`, should we keep it empty by default instead? From a consumer perspective I think it's better to get an explicit error if this attr is empty and then decide what values are valid instead of getting both families by default that might not apply to all use cases?